### PR TITLE
UX Icon: updated fa-exclamation-circle

### PR DESCRIPTION
**Pull request by NidhiKJha**
This PR is to update `fa-exclamation-circle` with `ICON_NAMES.WARNING`

#### Footer
Screenshot 2023-02-23 at 5 42 37 PM

#### Error Message
Screenshot 2023-02-23 at 5 42 54 PM

#### Footer
Screenshot 2023-02-23 at 5 42 16 PM

#### Error Message
Screenshot 2023-02-23 at 5 42 31 PM

## Pre-merge author checklist

- [ ] I've clearly explained:
  - [ ] What problem this PR is solving
  - [ ] How this problem was solved
  - [ ] How reviewers can test my changes
- [ ] Sufficient automated test coverage has been added

## Pre-merge reviewer checklist

- [ ] Manual testing (e.g. pull and build branch, run in browser, test code being changed)
- [ ] PR is linked to the appropriate GitHub issue
- [ ] **IF** this PR fixes a bug in the release milestone, add this PR to the release milestone

If further QA is required (e.g. new feature, complex testing steps, large refactor), add the `Extension QA Board` label.

In this case, a QA Engineer approval will be be required.

### CODE REVIEW

**Comment by darkwing** [comment]
I did a quick search of this icon and it looks like it's still used in `custom-spending-cap.js` and `review-spending-cap.js`

**Comment by Sylvia24** [comment]
We'd be happy to look into this for you. kindly file a support ticket below 
https://forms.gle/Wmpm6NLzYm9uXYuU9

File: ui/components/ui/review-spending-cap/review-spending-cap.js [hunk 1]
```diff
@@ -80,7 +80,10 @@ export default function ReviewSpendingCap({
                         fontWeight={FONT_WEIGHT.BOLD}
                         color={TextColor.errorDefault}
                       >
-                        <i className="fa fa-exclamation-circle" />{' '}
+                        <Icon
+                          name={ICON_NAMES.WARNING}
+                          style={{ verticalAlign: 'middle' }}
+                        />
```
Comment (NidhiKJha) [inline ui/components/ui/review-spending-cap/review-spending-cap.js:86]: ### Before
prev


### After
new

File: ui/components/app/custom-spending-cap/custom-spending-cap.js [hunk 2]
```diff
@@ -151,7 +151,7 @@ export default function CustomSpendingCap({
           fontWeight={FONT_WEIGHT.BOLD}
           color={TextColor.errorDefault}
         >
-          <i className="fa fa-exclamation-circle" /> {t('beCareful')}
+          <Icon name={ICON_NAMES.WARNING} /> {t('beCareful')}
```
Comment (NidhiKJha) [inline ui/components/app/custom-spending-cap/custom-spending-cap.js:154]: ### Before


pp

### After
b

**Review by brad-decker — APPROVED** [review]

**Review by darkwing — APPROVED** [review]

**Pull request merged**